site/app/controllers/grading/SimpleGraderController.php

```diff
@@ -36,7 +36,7 @@ public function printLab($gradeable_id, $section = null, $section_type = null, $
         elseif ($sort === "first") {
             $sort_by = "coalesce(NULLIF(u.user_preferred_givenname, ''), u.user_givenname)";
         }
-        else {
+        else if ($sort === "last") {
```

Comment:
could be good to have an else condition solely just so that future developers can see if their code changes made the $sort variable have an invalid value.